spec/acceptance/self_service_spec.rb

```diff
@@ -193,6 +193,10 @@ class {'self_service':
 - name: Classifier Configuration Data
   data_hash: classifier_data')
       end
+      it 'if S0031 conditions for false are met' do
+        result = run_shell('facter -p self_service.S0031')
+        expect(result.stdout).to match(%r{false})
```

Comment:
this test wont pass as you dont set your failure conditions, you will need to create some dummy folders

Reply:
Thanks for the review Marty, created a dummy directory in the test spec.